cli/command/service/list.go

```diff
@@ -40,6 +36,8 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Do not truncate the output")
```

Comment:
Same though as @thaJeztah, I don't really like `--no-trunc` flag, especially since format uses go template. We could provide a function that shorten "or not" the ID : `--format="{{ .ID | shorten }}"` or something like that.